Tasks-Lists/src/List_task_7/ListWithoutElements.java

```diff
@@ -0,0 +1,26 @@
+package List_task_7;
+import java.util.List;
+
+public class ListWithoutElements {
+    public static void main(String[] args) {
+
+    }
+     /* 7. Есть список с целыми числами. Написать функцию, которая вернет список без
+           элементов, больше заданного числа.
+           Если заданное число 3 -> {1, 2, 3, 4, 5} -> {1, 2}
+   */
+
+    public static List<Integer> removalElements(List<Integer> list, int n) {
+        for (int i = 0; i < list.size(); i++) {
+            if (list.get(i) >= n) {
+                list.remove(list.get(i));
```

Comment:
In this case you are changing the original input list, which is in general bad. It is better to create a new list and populate it with values from the original list, which pass the n value check.